src/main/groovy/life/qbic/datamodel/datasets/OxfordNanoporeMeasurement.groovy

```diff
@@ -70,12 +65,23 @@ final class OxfordNanoporeMeasurement {
     }
 
     private static void validateMetaData(Map metadata) throws IllegalArgumentException {
-        try {
-            MetaData.validateMetadata(metadata)
-        } catch (ValidationException e) {
-            // Aggregate the causing exceptions
-            def causes = e.getAllMessages().join("\n")
-            throw new IllegalArgumentException("The Nanopore metadata could not be collected.\nReason:\n$causes",)
+        def expectedKeys = [
+                "asic_temp",
+                "base_caller",
+                "base_caller_version",
+                "device_type",
+                "flow_cell_id",
+                "flow_cell_position",
+                "flow_cell_product_code",
+                "protocol",
+                "hostname",
+                "started"
+        ]
```

Comment:
This looks redundant to me, it's actually the same as defined in the METADATA_FIELD. What about using the enum and defining these strings there too. Then you can say the expected keys are the values defined in the enum METADATA_FIELD

Reply:
This is used to separate the way the information is provided (we can not influence that) from the way we process the metadata. Not all metadata that will be processed is required (adapter for example is not) so this needs to be a separate list.